### PR TITLE
chore: upgrade syft 0.36.0

### PR DESCRIPTION
**Pull request by developer-guy**
Signed-off-by: Batuhan Apaydın <batuhan.apaydin@trendyol.com>

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -6,9 +6,9 @@ LABEL org.opencontainers.image.source https://github.com/gythialy/golang-cross
 COPY entrypoint.sh /
 
 # install cosign
-COPY --from=gcr.io/projectsigstore/cosign:v1.4.1@sha256:502d5130431e45f28c51d2c24a05ef5ccd3fd916bcc91db0c8bee3a81e09a0bb /ko-app/cosign /usr/local/bin/cosign
+COPY --from=gcr.io/projectsigstore/cosign:v1.5.0@sha256:6ef3a23df63d53e90e821b2f71e0e5c45830d8dd5cc8a6fb8003fe50dba8605f /ko-app/cosign /usr/local/bin/cosign
 # install syft
-COPY --from=docker.io/anchore/syft:v0.34.0@sha256:ae6c51aae1de6e990e994b43943a7ce83823949dfe9e755cccd373e7deab3f8e /syft /usr/local/bin/syft
+COPY --from=docker.io/anchore/syft:anchore/syft:v0.36.0sha256:305e1777f6e105bfd4cc9a06faceefabb0a5c6c59d854013b4068c7ee7b310ba /syft /usr/local/bin/syft
```
Comment (cpanato) [inline Dockerfile:11]: i think you can remove the extra `anchore/syft:`


thanks for the pr!

**Review by cpanato — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (developer-guy) [reply Dockerfile]: thank you 🙈

**Comment by developer-guy** [comment]
IMO, we should wait for cosign 1.5.1 to merge it, so, I'll be prefixing the title with WIP

**Comment by cpanato** [comment]
we can bump only syft for now @developer-guy then we keep the pr smaller :)

**Comment by developer-guy** [comment]
> we can bump only syft for now @developer-guy then we keep the pr smaller :)

okay we can now merge this, I removed the changes related to cosign 🚀

**Review by cpanato — APPROVED** [review]
thanks

**Pull request merged**